app/view/fileupload/FileUploadWindow.js

```diff
@@ -0,0 +1,73 @@
+// define a simple Window class to hold the grid
```

Comment:
Change to [this style of comment](https://github.com/compassinformatics/cpsi-mapview/blob/master/app/view/window/MinimizableWindow.js#L1) with a view to generating autodocs in the future